ui/src/app/views/workspace/component.html

```diff
@@ -1,18 +1,18 @@
 <div *ngIf="!notFound else notFoundMessage">
   <div *ngIf="!workspaceLoading; else workspacePageLoading" style="display: flex;">
     <h2>{{workspace.name}}</h2>
-    <clr-button-group class="permissions-group btn-primary">
+    <clr-button-group class="permissions-group">
       <clr-button [disabled]="!writePermission" (click)="edit()"
           class="btn btn-icon">
-        <clr-icon class="is-solid" shape="pencil"></clr-icon>
+        <img src="assets/icons/edit@2x.png" alt="Edit" class="icon">
```

Comment:
We should add (minimally) a title attribute to each of these that contains the alt text if there's going to be no text on the button - this will allow discovery via hovering.